File: cmh/webapp/src/java/cl/cheekibreeki/cmh/webapp/bl/HoraDisponible.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package cl.cheekibreeki.cmh.webapp.bl;

import cl.cheekibreeki.cmh.lib.dal.entities.Bloque;
import java.util.Date;

/**
 *
 * @author dev
 */
public class HoraDisponible {
    private Date date;
    private int horaIni;
    private int horaFin;
    private int minuIni;
    private int minuFin;

    public HoraDisponible() {
    }

    public HoraDisponible(Date date, Bloque bloque) {
        this.horaIni = bloque.getNumHoraIni();
        this.horaFin = bloque.getNumHoraFin();
        this.minuIni = bloque.getNumMinuIni();
        this.minuFin = bloque.getNumMinuFin();
    }

    
    
    public Date getDate() {
        return date;
    }

    public void setDate(Date date) {
        this.date = date;
    }

    public int getHoraIni() {
        return horaIni;
    }

    public void setHoraIni(int horaIni) {
        this.horaIni = horaIni;
    }

    public int getHoraFin() {
        return horaFin;
    }

    public void setHoraFin(int horaFin) {
        this.horaFin = horaFin;
    }

    public int getMinuIni() {
        return minuIni;
    }

    public void setMinuIni(int minuIni) {
        this.minuIni = minuIni;
    }

    public int getMinuFin() {
        return minuFin;
    }

    public void setMinuFin(int minuFin) {
        this.minuFin = minuFin;
    }
    
}

File: cmh/terminal/CheekiBreeki.CMH.Terminal.BL/HoraDisponible.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using CheekiBreeki.CMH.Terminal.DAL;

namespace CheekiBreeki.CMH.Terminal.BL
{
    public class HoraDisponible
    {
        private DateTime date;
        private BLOQUE bloque;
        private int horaIni;
        private int horaFin;
        private int minuIni;
        private int minuFin;

        public DateTime Date
        {
            get { return date; }
            set { date = value; }
        }
        
        public BLOQUE Bloque
        {
            get { return bloque; }
            set { bloque = value; }
        }
        
        public int HoraIni
        {
            get { return horaIni; }
            set { horaIni = value; }
        }
        
        public int HoraFin
        {
            get { return horaFin; }
            set { horaFin = value; }
        }
        public int MinuIni
        {
            get { return minuIni; }
            set { minuIni = value; }
        }
        

        public int MinuFin
        {
            get { return minuFin; }
            set { minuFin = value; }
        }

        public HoraDisponible()
        {
        }

        public HoraDisponible(DateTime date, BLOQUE bloque)
        {
            this.horaIni = bloque.NUM_HORA_INI;
            this.horaFin = bloque.NUM_HORA_FIN;
            this.minuIni = bloque.NUM_MINU_INI;
            this.minuFin = bloque.NUM_MINU_FIN;
            this.bloque = bloque;
        }
    }
}
